### ISSUE TITLE
How to add custom shaders to Dolphin emulator

### ISSUE THREAD

**Issue by jerkstore369** [libretro/common-shaders#159]
Is it possible to add a CRT shader to Dolphin emulator? I need it to look like when I played them back in the day on a CRT but I don't know much about computers. What are all these files? Is there a file you can just drag and drop somewhere? Thanks.

**Comment by hizzlekizzle** [comment]
Yes, RetroArch comes with many of them. If you get the dolphin-libretro core running in RetroArch, you can load a shader from the quick menu to apply it. The only catch is that most of them expect a low-res image as input, so they typically look bad if you increase the internal resolution a bunch.
